lib/firebase_storage.js

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
@anonrig looks useful! I assume that you don't mean just for this file but for the whole codebase, right?